files/iframe-resizer/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "iframe-resizer",
+  "repo": "davidjbradshaw/iframe-resizer",
+  "files": {
+    "basePath": "js"
```

Comment:
Add an `include` parameter with `*.min.*`
We dont need all the files in js/ for production